src/components/valet-reservation/CurrentSpots.jsx

```diff
@@ -1,55 +1,25 @@
-  import { useState, useEffect, useMemo } from "react";
-  import { useSelector, useDispatch } from "react-redux";
-  import { setTime } from "../../utils/slice/valetSlice";
-  import getSpotMetrics from "../../utils/getSpotMetrics";
+  import { useEffect } from "react";
+  import { useSelector } from "react-redux";
 
   const CurrentSpots = () => {
-    //will implement useMemo and redux later
-    const { time } = useSelector((state) => state.valet);
-    const dispatch = useDispatch();
-    const [availableSpots, setAvailableSpots] = useState(0);
-    const [reservedSpots, setReservedSpots] = useState(0);
-    const [occupiedSpots, setOccupiedSpots] = useState(0);
-
-
-    const fetchSpots = async () => {
-      //hardcoding data for now
-      const garageId = 1;
-      const date = "12-28-23";
-      const time = 6;
-      try {
-        const data = await getSpotMetrics(garageId, date, time);
-        const { available, reserved, occupied } = data;
-        setAvailableSpots(available);
-        setReservedSpots(reserved);
-        setOccupiedSpots(occupied);
-      } catch (error) {
-        console.log(error);
-      }
-    };
-
-    useEffect(() => {
-      fetchSpots();
-    }, []);
+    const { date, spots } = useSelector((state) => state.valet);
 
```

Comment:
What is the purpose of this useEffect, is it just for troubleshooting purposes?

Reply:
Yes, just for troubleshooting. Will remove later